### ISSUE TITLE
Interpreting values as their definitions in the DBC

### ISSUE THREAD

**Issue by wakey1991** [danielhrisca/asammdf#1046]
I am logging a series of signals, some of which show a number value like ambient air temperature and vehicle speed however i also have some that show multiple states such as 0 = inactive, 1 = active, 2 = error, 3 = not available. I cannot seem to find a way for ASAMMDF to show the signal as its interpretation in the DBC. ASAMMDF will just show the number.

Apologies if i am overlooking something obvious, i cannot find it in the help file.

**Comment by danielhrisca** [comment]
Are you decoding a raw CAN bus logging mf4 file using a DBC?

**Comment by wakey1991** [comment]
Yes MF4 and then extract bus signals using a DBC file built in KVASER database editor

**Comment by danielhrisca** [comment]
Perhaps you should try the development branch code

**Comment by wakey1991** [comment]
I am not familiar with that, is there not a simple setting i can change to support this?

**Comment by danielhrisca** [comment]
Pip install https://github.com/danielhrisca/asammdf/archive/development.zip

**Comment by wakey1991** [comment]
I have downloaded this but even following the readme it isnt clear what i do with these files in order to make my request work?

My skill is in CAN but not so good with computer programming.

**Comment by wakey1991** [comment]
> Pip install https://github.com/danielhrisca/asammdf/archive/development.zip

@danielhrisca ive done some researched and i assume from this i need to install python? I have no experience with this at all

Is it something you could advise with, will this be a simple installation?

I just would like my CAN signals to have definitions.

**Comment by danielhrisca** [comment]
you can try the bundled build found here https://github.com/danielhrisca/asammdf/actions/runs/10143948078 (dist archive)

**Comment by wakey1991** [comment]
Hi, i downloaded and extracted but the program does not run.

I get the following errors
![Screenshot 2024-07-30 212332](https://github.com/user-attachments/assets/7bc34c30-37d9-4b49-a651-f886f826b53e)
2024-07-30, 21:23:22
--------------------------------------------------------------------------------
<class 'AttributeError'>	 
partially initialized module 'fsspec' has no attribute 'utils' (most likely due to a circular import)
--------------------------------------------------------------------------------
  File "asammdf\gui\asammdfgui.py", line 50, in <module>
  File "asammdf\gui\asammdfgui.py", line 43, in main
  File "asammdf\gui\widgets\main.py", line 47, in __init__
  File "asammdf\gui\widgets\batch.py", line 121, in __init__
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "fastparquet\__init__.py", line 4, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "fastparquet\writer.py", line 12, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "fastparquet\util.py", line 16, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "fsspec\__init__.py", line 6, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "fsspec\compression.py", line 68, in <module>
  File "fsspec\compression.py", line 46, in register_compression

![Screenshot 2024-07-30 212439](https://github.com/user-attachments/assets/b8f246e7-a729-4b86-a435-351a18174959)

I am not a programmer so this doesn't mean anything to me, is it something my end that is wrong or could i have installed it incorrectly?
